source/_posts/2017/2017-09-04-how-apigen-survived-its-death.md

```diff
@@ -0,0 +1,202 @@
+---
+id: 53
+layout: post
+title: "How ApiGen Survived its Death"
```

Comment:
I would change it to "How ApiGen Survived Its Own Death?"

Reply:
Nice point!